packages/chargebee/src/checkout/index.stories.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Checkout from './index';
+
+export default { title: 'Checkout', component: Checkout };
+
+// Reference: https://www.chargebee.com/checkout-portal-docs/api.html#chargebee-instance-object
+const dummyData = {
```

Comment:
Examples are not really working for me - is the dummy data correct?

Reply:
No it does not. Just useful to know the structure of the payload

Reply:
explain in the comment that this is an example output